backend/stakepoold/rpc/rpcserver/context.go

```diff
@@ -53,6 +53,7 @@ type AppContext struct {
 	WalletConnection       *rpcclient.Client
 	WinningTicketsChan     chan WinningTicketsForBlock
 	Testing                bool // enabled only for testing
+	ColdWalletExtPub       string
```

Comment:
I'll prefer this be moved to after line 42 and similarly assigned after line 204 in `backend/stakepoold/server.go`. Both are simple copies of config values from the `cfg` object to the `AppContext` instance; keeping them close together makes it slightly easier to read/understand imo.